### ISSUE TITLE
[PROJECT PROPOSAL]

### ISSUE THREAD

**Issue by kimix7** [CognitiveLab-tech/World-of-AI#5]
## Project Request

Develop an object detection model using YOLO. An image will be the input to the model and the model will broadly classify the image into the classes it has been trained on. 

---

| Field  | Deep Learning                       |
| ------ | --------------------------------- |
| About  | An object detection model using YOLO |
| Github | kimix7                  |
| Email  | kimayashejwalkar8@gmail.com                                  |
| Label  | Project Request                   |

https://github.com/kimix7

---

**Define You**

- [ ] GSSOC Participant
- [ ] Contributor

# Object detection using YOLO

## Description
The object detection project using YOLO (You Only Look Once) focuses on utilizing deep learning techniques to detect and track various objects in images or real-time video streams. By training a YOLO model with a dataset that includes labelled images, the model learns to recognize and localize objects within new images or video frames. This project provides accurate and efficient detection of objects, including people.

## Scope

Objectives:
- Implement a robust and efficient object detection system using YOLO.
- Enable accurate detection and localization of various objects, including specific target objects.
- Learning how YOLO works.

Deliverables
- A trained YOLO model capable of detecting and localizing objects accurately.
- Well-documented guidelines, including dataset preparation, training, inference, and any specific requirements.
- Efficient and real-time object detection on images or video frames.

Constraints
- Availability and quality of labelled training data can impact the model's performance.
- Achieving high accuracy requires sufficient training iterations and careful fine-tuning of the model's parameters. It may involve iterative experimentation and adjustment to optimize detection performance.

## Timeline
Start Date: when assigned
End Date: 10 August

**Comment by lakshmi-n-i** [comment]
I'm very interested in working on this, please assign this to me under GSSC'23

**Comment by samarthjain422005** [comment]
i amm intereted in this  can you please assign it to me under gssoc

**Comment by tushtithakur** [comment]
Hi, I am very much interested and would like to contribute as a part of @GSSOC'23. 
Please assign the task to me. Thank you.

**Comment by samarthjain422005** [comment]
I'm interested in working on this project, please assign this to me under GSSOC'23

**Comment by adithya-s-k** [comment]
@kimix7  I have assigned this particular project to you under GSSOC 23.
Will you be able to manage the project alone or would you like to collaborate with other contributors

All the best contributing
if you have any doubts on the project structure feel free to contact me

**Comment by adithya-s-k** [comment]
@samarthjain422005 @lakshmi-n-i @tushtithakur 
guys I will be creating more beginner to expert level issues soon
please keep an eye out for that.

**Comment by kimix7** [comment]
> @kimix7  I have assigned this particular project to you under GSSOC 23.
> Will you be able to manage the project alone or would you like to collaborate with other contributors
> 
> All the best contributing
> if you have any doubts on the project structure feel free to contact me

I would like to try to do it alone but would like some help of I am stuck somewhere.

**Comment by kimix7** [comment]
> @kimix7 I have assigned this particular project to you under GSSOC 23. Will you be able to manage the project alone or would you like to collaborate with other contributors
> 
> All the best contributing if you have any doubts on the project structure feel free to contact me

sure will.

**Comment by mehak1309** [comment]
I am a GSSoC'contributor. I am interested in this project. Could you please assign this to me?
